src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,103 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int INCREASE_COEFFICIENT = 2;
+    private Node<K,V>[] hashMapTable;
+    private int size;
+
+    public MyHashMap() {
+        hashMapTable = (Node<K, V>[]) new Node[DEFAULT_CAPACITY];
```

Comment:
має працювати без цього
```suggestion
        hashMapTable = new Node[DEFAULT_CAPACITY];
```